pkg/clusters/addons/kong/kong.go

```diff
@@ -29,14 +29,21 @@ const (
 	AddonName clusters.AddonName = "kong"
 
 	httpPort = 80
+
+	// EnterpriseImageRepo default kong enterprise image
+	EnterpriseImageRepo = "kong/kong-gateway"
+
+	// EnterpriseImageTag latest kong enterprise image tag
+	EnterpriseImageTag = "2.5.0.0-alpine"
```

Comment:
Given that we intend to make these values changeable in the `Builder`, it seems we should maybe name these with a `Default*` prefix to indicate that they are the defaults if nothing else is provided:

```suggestion

	// DefaultEnterpriseImageRepo default kong enterprise image
	DefaultEnterpriseImageRepo = "kong/kong-gateway"

	// DefaultEnterpriseImageTag latest kong enterprise image tag
	DefaultEnterpriseImageTag = "2.5.0.0-alpine"
```

Reply:
67c9bddf4a2fc7aaa96ca14ad3e1537d68fa473d